src/graph/backend/dnnl/kernels/sdp_decomp_config.cpp

```diff
@@ -229,6 +229,25 @@ impl::status_t sdp_decomp_config_t::construct_params(
     // softmax
     // create softmax primitive attr
     dnnl::primitive_attr sub_softmax_attr = make_primitive_attr(sdp_op[2], mgr);
+
+    dnnl_pops = {};
+    ori_dnnl_pops = sub_softmax_attr.get_post_ops();
+    for (int i = 0; i < ori_dnnl_pops.get()->len(); i++) {
+        auto alg = static_cast<algorithm>(
```

Comment:
always add const unless you expect it will be changed later:  alg, post_shape, post_stride etc.
```suggestion
        const auto alg = static_cast<algorithm>(
```

Reply:
Added, thanks.